### PR TITLE
Reduced wizard background image opacity using an overlay

### PR DESCRIPTION
**Pull request by rb1193**
No idea why I couldn't figure out how to do this first time around, it's not exactly difficult. Anyway, this should make the content a lot easier to read. There's a minor responsiveness improvement for the options as well.
___
NB - by making a pull request you confirm agreement to the [Contributors Agreement](https://github.com/openlawnz/openlawnz-web/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: src/components/Wizard/Wizard.jsx [hunk 1]
```diff
@@ -26,11 +26,16 @@ export default function Wizard({ background, title, steps }) {
...
-        <WizardStart title={title} start={() => dispatch({ type: WizardOperations.NEXT, payload: steps[0].key })} />
-      ) : (
-        <WizardStep step={steps.find((step) => step.key === wizardStack[wizardStack.length - 1])} navigate={dispatch} />
-      )}
+      <div className="wizard-bg-overlay">
+        {wizardStack.length === 0 ? (
+          <WizardStart title={title} start={() => dispatch({ type: WizardOperations.NEXT, payload: steps[0].key })} />
+        ) : (
+          <WizardStep
+            step={steps.find((step) => step.key === wizardStack[wizardStack.length - 1])}
```
Comment (danieljimeneznz) [inline src/components/Wizard/Wizard.jsx:34]:
```suggestion
            step={steps.find(({ key }) => key === wizardStack[wizardStack.length - 1])}
```

**Comment by BenRauzi** [comment]
Are you able to make that change to keep the 'options' at a fixed height in this PR?

**Comment by rb1193** [comment]
> Are you able to make that change to keep the 'options' at a fixed height in this PR?

I had been holding off because I thought Will and Ying were still debating about it... can do though if it's definitely wanted

**Comment by rb1193** [comment]
@BenRauzi may I suggest we merge this for now and we'll pick up any minor tweaks such as the options spacing at a later date? Don't want to get too far behind the dev branch

**Pull request merged**